pages/trust-admin/hospitals/add.js

```diff
@@ -111,49 +68,12 @@ const AddAHospital = ({ error, trustId }) => {
       <GridRow>
         <GridColumn width="two-thirds">
           <ErrorSummary errors={errors} />
-          <form onSubmit={onSubmit}>
-            <Heading>Add a hospital</Heading>
-            <FormGroup>
-              <Label htmlFor="hospital-name" className="nhsuk-label--l">
-                What is the hospital name?
-              </Label>
-              <Input
-                id="hospital-name"
-                type="text"
-                hasError={hasError("hospital-name")}
-                errorMessage={errorMessage("hospital-name")}
-                className="nhsuk-u-font-size-32"
-                style={{ padding: "16px!important", height: "64px" }}
-                onChange={(event) => setHospitalName(event.target.value)}
-                name="hospital-name"
-                autoComplete="off"
-                value={hospitalName || ""}
-              />
-            </FormGroup>
-
-            <FormGroup>
-              <Label htmlFor="hospital-survey-url" className="nhsuk-label--l">
-                Key contact survey URL (optional)
-              </Label>
-              <span className="nhsuk-hint" id="hospital-survey-url-hint">
-                The survey URL will appear on the visit complete page for the
-                key contact.
-              </span>
-              <Input
-                id="hospital-survey-url"
-                type="url"
-                hasError={hasError("hospital-survey-url")}
-                errorMessage={errorMessage("hospital-survey-url")}
-                className="nhsuk-u-font-size-32"
-                style={{ padding: "16px!important", height: "64px" }}
-                onChange={(event) => setHospitalSurveyUrl(event.target.value)}
-                name="hospital-survey-url"
-                value={hospitalSurveyUrl || ""}
-              />
-            </FormGroup>
-
-            <Button className="nhsuk-u-margin-top-5">Add hospital</Button>
-          </form>
+          <EditHospitalForm
+            errors={errors}
+            setErrors={setErrors}
+            hospital={{}}
+            submit={submit}
+          />
```

Comment:
🎉

Reply:
Nitpick: I think if we update `EditHospitalForm` to `const EditHospitalForm = ({ errors, setErrors, hospital = {}, submit }) => {` with a default value, it'll meet that we won't have to do `hospital={{}}`.

Reply:
Good spot, will do that.